core/src/main/java/org/kiwix/kiwixmobile/core/extensions/ViewExtensions.kt

```diff
@@ -69,3 +71,11 @@ fun View.snack(
     })
   }.show()
 }
+
+fun View.setToolTip(description: String) {
+  if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+    TooltipCompat.setTooltipText(this, description)
+  } else {
+    contentDescription = description
```

Comment:
@Sagar0-0 This code is not working with API level 24. It does not show any title while long clicking on the close all tabs button. However, `TooltipCompat.setTooltipText(this, description)` this method automatically handles the for both above and below API level 26. You should use this method it will handle the backward compatibility.

Reply:
Resolved